app/models/user.js

```diff
@@ -11,7 +11,7 @@ module.exports = (sequelize, DataTypes) => {
         type: DataTypes.STRING,
         allowNull: false
       },
-      lastName: {
+      last_name: {
```

Comment:
Why? models should be in camelCase, please check the best practices cards please :)